lib/config/configuration.go

```diff
@@ -1208,17 +1209,33 @@ func applyWindowsDesktopConfig(fc *FileConfig, cfg *service.Config) error {
 	}
 	ldapPassword, err := os.ReadFile(fc.WindowsDesktop.LDAP.PasswordFile)
 	if err != nil {
-		return trace.WrapWithMessage(err, "loading LDAP password from file %v",
+		return trace.WrapWithMessage(err, "error loading LDAP password from file %v",
 			fc.WindowsDesktop.LDAP.PasswordFile)
 	}
+
+	var cert *x509.Certificate
+	if !fc.WindowsDesktop.LDAP.SkipVerifyCA {
+		raw_cert, err := ioutil.ReadFile(fc.WindowsDesktop.LDAP.CAFile)
+		if err != nil {
+			return trace.WrapWithMessage(err, "error loading LDAP root CA from file %v", fc.WindowsDesktop.LDAP.CAFile)
```

Comment:
```suggestion
			return trace.WrapWithMessage(err, "error loading LDAP CA from file %v", fc.WindowsDesktop.LDAP.CAFile)
```

It's not necessarily a root CA, could be an intermediate or a full chain, so I would just omit "root".